packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -1051,6 +1051,7 @@ export class EksOptimizedImage implements ec2.IMachineImage {
     this.amiParameterName = `/aws/service/eks/optimized-ami/${this.kubernetesVersion}/`
       + ( this.nodeType === NodeType.STANDARD ? 'amazon-linux-2/' : '' )
       + ( this.nodeType === NodeType.GPU ? 'amazon-linux-2-gpu/' : '' )
+      + (this.nodeType === NodeType.INFERENTIA ? 'amazon-linux-2-gpu/' : '')
```

Comment:
Why do we need a separate node type here? Can't we just identify `inf1` instances as a gpu?

Reply:
We just announced the **Amazon ECS-optimized Amazon Linux 2 (Inferentia) AMI**, which has different parameter store path and AMI value from the GPU one.  I guess we probably will have similar AMI branch for Amazon EKS

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html

Reply:
There're some device plugins such as

https://github.com/NVIDIA/k8s-device-plugin
and
https://github.com/awslabs/aws-virtual-gpu-device-plugin

And I agree we hard-code it for this support to make it simple.